src/Entity/Adherent.php

```diff
@@ -1622,6 +1622,14 @@ public function getMunicipalChiefManagedArea(): ?MunicipalChiefManagedArea
         return $this->municipalChiefManagedArea;
     }
 
+    public function setMunicipalChiefManagedAreaFromArray(array $municipalChiefManagedAreas): void
+    {
+        $municipalChiefManagedArea = new MunicipalChiefManagedArea();
```

Comment:
why not directly?
```
$this->municipalChiefManagedArea = new MunicipalChiefManagedArea();
...
```

Reply:
In fact, at the moment i typd this code i thought this would be clearer, but you are right, this is not necessarry.

Reply:
In fact, at the moment i typd this code i thought this would be clearer, but you are right, this is not necessarry.